misc minor changes v0.9.1 sander

sanderclaeys opened the pull request:
- ADD: added a transformation `rm_bounds!` that removes all fields ending in `_ub` and `_lb`
- FIX: added missing `connections` for virtual generator at voltage source
- FIX: corrected pu conversion bus voltage bounds, and added parsing for `vm_pair_lb` and `vm_pair_ub`
- STY: corrected indent level

pseudocubic commented:
The scope of your original PR was perfect; it was the addition of a single function related to one specific feature you were looking for (eliminating default bounds), and didn't touch unrelated parts of the code. It is better to have single issue PRs and merge them as we go; it has a lot of advantages including better tracking of new features, ensure that all changes are represented in the changelog, easier code reviews, etc. I will review this PR as it is now, but in the future unrelated changes should be in separate PRs to make things much more manageable.

Also, in the next few days I will merge #276, add some default Github Checks to PRs, and PRs will be expected to have certain things, such as unit tests, CHANGELOG updates, etc., which will be enumerated in a PR template so that expectations are clear. If you could review the documentation added by that PR and update this PR accordingly that would be great. 

A final note is that this can be merged directly to master; only breaking changes really need to be merged to a release candidate branch for a project of this size. We can change that before this PR gets merged though, so its not really an issue, just something to take note of for the future.

sanderclaeys commented:
In the future I will create separate PRs for this. For this one, I 
- added prefixes as requested by the template to my initial bullet list,
- added entries to the CHANGELOG,
- added a unit test for `rm_bounds!`.

I think the title already gives sufficient information about the individual changes, so I did not add a body to the PR beyond what I wrote initially. Does this conform with the new requirements?

pseudocubic commented on src/data_model/transformations.jl:
```diff
@@ -47,3 +47,19 @@ function apply_voltage_bounds!(data_eng::Dict{String,<:Any}; vm_lb::Union{Real,M
 end
+
+
+"removes all fields ending in '_ub' or '_lb'"
+function rm_bounds!(data_eng)
+    for (k,v) in data_eng
```
I suggest renaming this function to be more verbose, like `remove_all_bounds!`; I think we will want different variants of transformation functions that remove bounds, therefore keeping the names clear will be extra important

pseudocubic commented:
@sanderclaeys I think we are only waiting for a function rename for me to merge this

sanderclaeys commented:
My mistake my commit did not get pushed because of access rights in my git client

pseudocubic commented:
It would be better to use `remove` instead of `rm` in the function name, see the subsection of Functions under the section Style Conventions in the CONTRIBUTING guide
